Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/NoHardCodedPathsAnalyzer.cs

```diff
@@ -18,10 +18,11 @@ namespace Philips.CodeAnalysis.MaintainabilityAnalyzers.Maintainability
 	[DiagnosticAnalyzer(LanguageNames.CSharp)]
 	public class NoHardCodedPathsAnalyzer : SingleDiagnosticAnalyzer
 	{
+		private const int MaxStringLength = 1000;
```

Comment:
Please remove note about Linux; suggest replaces with note that this analyzer is Windows only.

Reply:
Done